kinde_sdk/management/schemas.py

```
@@ -10,23 +10,23 @@
-from collections import defaultdict
-from datetime import date, datetime, timedelta  # noqa: F401
-import functools
 import decimal
+import functools
 import io
 import re
 import types
 import typing
 import uuid
+from collections import defaultdict
+from datetime import date, datetime, timedelta  # noqa: F401
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)